### ISSUE TITLE
check_snmp_metric not working on 2.2

### ISSUE THREAD

**Issue by dnlldl** [HeinleinSupport/check_mk_extensions#132]
When enabled, the plugin doesn't work and make the checkmk config crash.

```
Traceback (most recent call last):
  File "/omd/sites/home/lib/python3/cmk/update_config/main.py", line 223, in __call__
    action(self._logger, self.update_state.setdefault(action.name))
  File "/omd/sites/home/lib/python3/cmk/update_config/plugins/actions/rulesets.py", line 54, in __call__
    _transform_wato_rulesets_params(
  File "/omd/sites/home/lib/python3/cmk/update_config/plugins/actions/rulesets.py", line 160, in _transform_wato_rulesets_params
    valuespec = ruleset.valuespec()
                ^^^^^^^^^^^^^^^^^^^
  File "/omd/sites/home/lib/python3/cmk/gui/watolib/rulesets.py", line 879, in valuespec
    return self.rulespec.valuespec
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/omd/sites/home/lib/python3/cmk/gui/watolib/rulespecs.py", line 348, in valuespec
    return self._valuespec()
           ^^^^^^^^^^^^^^^^^
  File "<string>", line 100, in _valuespec_active_checks_snmp_metric
```

**Comment by gurubert** [comment]
The end of the traceback is missing. Please add it.

**Comment by gurubert** [comment]
should be fixed in version 0.4.3 of the check_snmp_metric extension

**Comment by dnlldl** [comment]
> The end of the traceback is missing. Please add it.
I'm surprised anything was missing. My bad.

> should be fixed in version 0.4.3 of the check_snmp_metric extension
I can confirm it now works! Thank you.
